src/codeyCommand.ts

```diff
@@ -47,13 +47,11 @@ export class CodeyCommand extends SapphireCommand {
   }
 
   // Regular command
-  public async messageRun(message: Message): Promise<Message<boolean>> {
+  public async messageRun(message: Message): Promise<Message<boolean> | undefined> {
     const { client } = container;
-    const initialMessageFromBot: SapphireMessageRequest = await message.channel.send({
-      content: this.messageWhenExecutingCommand
-    });
     try {
-      const successResponse = await this.executeCommand(client, message, initialMessageFromBot);
+      const successResponse = await this.executeCommand(client, message);
+      if (!successResponse) return;
```

Comment:
if I do return here, what happens?
can we just throw instead for err msg?

Reply:
For example, in the `ping` command, the edit of the `Ping?` message already occurs inside `PingCommand`, so nothing needs to be sent from the abstraction of `CodeyCommand`. This is signified through `return Promise.resolve('');`, so `!successResponse` evaluates to true, effectively returning from the function and doing nothing more. This is the expected behaviour, so no error is thrown.